modules/codegen/src/test/scala/core/issues/Issue313.scala

```diff
@@ -0,0 +1,92 @@
+package core.issues
+
+import cats.data.NonEmptyList
+import com.twilio.guardrail.generators.Scala.AkkaHttp
+import com.twilio.guardrail.generators.syntax.Scala.companionForStaticDefns
+import com.twilio.guardrail.{ Client, Clients, Context }
+import org.scalatest.funsuite.AnyFunSuite
+import org.scalatest.matchers.should.Matchers
+import support.SwaggerSpecRunner
+
+import scala.meta._
+
+class Issue313 extends AnyFunSuite with Matchers with SwaggerSpecRunner {
+  val swagger: String = s"""
+    |openapi: 3.0.2
+    |info:
+    |  title: Whatever
+    |  version: 1.0.0
+    |servers:
+    |  - url: http://localhost:1234
+    |paths:
+    |  /users/{name}:
+    |    patch:
+    |      operationId: changeUser
+    |      parameters:
+    |      - name: name
+    |        in: path
+    |        required: true
+    |        schema:
+    |          type: string
+    |      requestBody:
+    |        required: true
+    |        content:
+    |          multipart/form-data:
+    |            schema:
+    |              required:
+    |                - id
+    |              properties:
+    |                id:
+    |                  type: integer
+    |                  format: int64
```

Comment:
I thought this bug only triggered when `type: object` is explicitly specified. At least my workaround was always to omit that...

Reply:
I ran into this again with the Twilio OpenAPI specs -- it should be possible to generate clients and servers directly from the specifications, unmodified.

Reply:
Right, but I mean that this example spec here always worked fine for me; that is, even before this fix, there was no extra body parameter present.  To trigger the bad behavior I always had to add `type: object`.  I think there was actually a sample where I commented it out with the hopes that we'd fix it someday.  Right, yeah, here: https://github.com/guardrail-dev/guardrail/blob/master/modules/sample/src/main/resources/multipart-form-data.yaml#L17-L22

(It's also possible that I hackily "fixed" this in the Java server & client generators by ignoring `bodyParams` if `formParams` was non-empty.)

Reply:
Yeah -- Removing `type: object` is the workaround for this issue, but manually modifying generated OpenAPI specifications is untenable.

With this patch, that comment is resolved, since it prevents the `ObjectSchema` from being processed in the case of form. I tried generating AsyncHttpClient, fwiw, and the useless `body` was emitted for that as well, so if there is a workaround hidden in the Java code, it's not in the client generator.